examples/async/.babelrc

```diff
@@ -1,19 +1,20 @@
 {
-  "stage": 2,
+  "presets": ["es2015","stage-2", "react"],
   "env": {
     "development": {
       "plugins": [
-        "react-transform"
-      ],
-      "extra": {
-        "react-transform": {
-          "transforms": [{
-            "transform": "react-transform-hmr",
-            "imports": ["react"],
-            "locals":  ["module"]
-          }]
-        }
-      }
+        [
+          "react-transform", {
+            "transforms": [{
+              "transform": "react-transform-hmr",
+              "imports": ["react"],
+              "locals": ["module"]
+            }]
+          }
+        ]
+      ]
     }
   }
-}
+
+
+}
```

Comment:
Can we remove the blank lines that have been introduced into the package.json files in the examples please?